templates/modules/hero.html

```diff
@@ -36,7 +36,7 @@
         <div class="mx-auto flex h-full max-w-7xl flex-col items-center justify-center gap-3 px-4 py-6 lg:px-6">
           <span class="text-5xl" th:style="|color:${theme.config.layout.title_color}|" th:text="${site.title}"></span>
           <span
-            class="text-sm font-light opacity-50"
+            class="text-sm font-light opacity-100"
```

Comment:
如果不设置透明度的话，直接移除这个 class 就行了。

```suggestion
            class="text-sm font-light"
```